composer.json

```diff
@@ -13,7 +13,8 @@
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^1.1",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "friendsofphp/php-cs-fixer": "~2"
```

Comment:
Sure, nevertheless this allows people to provide consistent CS on PR they submit.
Also projects require this library use the `--no-dev` during the build process, so this dependencies will be excluded.

In this case in might be complicate for a developer to install the right version of php-cs-fixer (~1 and ~2) are not compatibles, and not force them to install a specific version globally